public_domain_review/utils.py

```diff
@@ -0,0 +1,45 @@
+import logging
+from urllib.parse import urljoin
+
+import requests
+from bs4 import BeautifulSoup
+
+SOURCE_NAME = "public-domain-review"
+
+
+def get_outbound_links(url, html):
+    soup = BeautifulSoup(html, "html.parser")
+    links = soup.find_all("a")
+    outbound_links = [
+        urljoin(url, link.get("href")) for link in links if link.get("href")
+    ]
+    return set(outbound_links)
+
+
+def get_content(url):
```

Comment:
We have a unified getter function in `licensed_pile/scrap.py` which does things like retrys and sets a nicer user-agent. Lets move to that instead of this function.